src/installer.py

```diff
@@ -82,12 +82,19 @@ def dlstatus():
         trd1.start()
         trd2.start()
 
-        #reboot
+        #Start Waydroid Container
         child.expect('root.*', timeout=10000)
-        print("reboot")
-        pyotherside.send('whatState',"=> rebooting")
-        child.sendline("reboot")
+        print("starting waydroid container")
+        pyotherside.send('whatState',"=> starting waydroid container")
+        child.sendline("sudo service waydroid-container start")
```

Comment:
This is no longer needed, `waydroid-container` is started after install [since Waydroid v1.3.3 UT 16.04 packaging](https://gitlab.com/ubports/development/core/packaging/waydroid/-/commit/2b7636fa76823f8d26355a01020328e62ad2d70f) :)

Reply:
Oooh

Reply:
I still see this being manually started in the latest revision :p

Reply:
You're too quick :p